Code/Core/Env/glibc_compat_stat.h

```diff
@@ -0,0 +1,39 @@
+// glibc_compat_stat
```

Comment:
Named this file in camel_case to match existing glibc_compat.h.
But maybe I should have named it PascalCase to match other header files? (since it is included directly)